### ISSUE TITLE
pptv 不能解析

### ISSUE THREAD

**Issue by kungfucop** [soimort/you-get#2372]
- [x] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [x ] 您的`you-get`为最新版.
- [ x] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [x ] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [ x] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

$ you-get 'http://v.pptv.com/show/RYmCAczxibjiabGYE.html' --debug
[DEBUG] get_content: http://v.pptv.com/show/RYmCAczxibjiabGYE.html
[DEBUG] get_content: http://web-play.pptv.com/webplay3-0-26611269.xml?type=web.fpp&version=4
site:                PPTV
title:               刮断一根树枝索赔十八万
stream:
    - format:        1
      container:     mp4
      video-profile: 720x404@601kbps
      size:          3.6 MiB (3764280 bytes)
    # download-with: you-get --format=1 [URL]

Downloading 刮断一根树枝索赔十八万.mp4 ...
 0.0% (  0.0/  3.6MB) ├───────────────────────────────────────────────────────────────────────────────────────────────────────────────┤[1/1] [DEBUG] HTTP Error with code403
[DEBUG] HTTP Error with code403
you-get: version 0.4.915, a tiny downloader that scrapes the web.
you-get: Namespace(URL=['http://v.pptv.com/show/RYmCAczxibjiabGYE.html'], cookies=None, debug=True, extractor_proxy=None, force=False, format=None, help=False, http_proxy=None, info=False, input_file=None, itag=None, json=False, no_caption=False, no_merge=False, no_proxy=False, output_dir='.', output_filename=None, password=None, player=None, playlist=False, socks_proxy=None, stream=None, timeout=600, url=False, version=False)
Traceback (most recent call last):
  File "/usr/local/bin/you-get", line 11, in <module>
    load_entry_point('you-get==0.4.915', 'console_scripts', 'you-get')()
  File "/usr/local/lib/python3.6/site-packages/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 1362, in main
    script_main(any_download, any_download_playlist, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 1270, in script_main
    **extra
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 1072, in download_main
    download(url, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 1355, in any_download
    m.download(url, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/extractor.py", line 59, in download_by_url
    self.download(**kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/extractor.py", line 227, in download
    av=stream_id in self.dash_streams)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 785, in download_urls
    url_save(url, output_filepath, bar, refer = refer, faker = faker, headers = headers, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 533, in url_save
    file_size = url_size(url, faker=faker, headers=tmp_headers)
  File "/usr/local/lib/python3.6/site-packages/you_get/common.py", line 439, in url_size
    size = response.headers['content-length']
AttributeError: 'NoneType' object has no attribute 'headers'```
[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]
> [舔 :icecream:!]

**Comment by rosynirvana** [comment]
```
./you-get -d http://v.pptv.com/show/RYmCAczxibjiabGYE.html -o /tmp
[DEBUG] get_content: http://v.pptv.com/show/RYmCAczxibjiabGYE.html
[DEBUG] get_content: http://web-play.pptv.com/webplay3-0-26611269.xml?type=web.fpp&version=4
site:                PPTV
title:               刮断一根树枝索赔十八万
stream:
    - format:        1
      container:     mp4
      video-profile: 720x404@601kbps
      size:          3.6 MiB (3764280 bytes)
    # download-with: you-get --format=1 [URL]

Downloading 刮断一根树枝索赔十八万.mp4 ...
 100% (  3.6/  3.6MB) ├█████████████████████████████████████████┤[1/1]    2 MB/s
```

**Comment by kungfucop** [comment]
thanks @rosynirvana  it might because of I am not in China?

**Comment by seahog004** [comment]
@rosynirvana 你好，我这里所有PPTV链接解析出来都是同一个广告。

you-get -d http://v.pptv.com/show/RYmCAczxibjiabGYE.html -o F:
[DEBUG] get_content: http://v.pptv.com/show/RYmCAczxibjiabGYE.html
[DEBUG] get_content: http://web-play.pptv.com/webplay3-0-26611269.xml?type=web.f
pp&version=4
site:                PPTV
title:               国外一支超欢脱广告，全程脑洞
stream:
    - format:        0
      container:     mp4
      video-profile: 480x272@328kbps
      size:          3.3 MiB (3494173 bytes)
    # download-with: you-get --format=0 [URL]

Downloading 国外一支超欢脱广告，全程脑洞.mp4 ...
 0.0% (  0.0/  3.3MB) ├────────────────────────────
─────────────┤[1/1]

**Comment by airdge** [comment]
`http://web-play.pptv.com/webplay3-0-26611269.xml?type=web.fpp&version=4&appplt=flp&appid=pptv.flashplayer.vod&appver=3.4.2.12`

参数缺失,获取的资源不对,带上&appplt=flp&appid=pptv.flashplayer.vod&appver=3.4.2.12参数就行
将原pptv.py里的201行
` api_url += '?type=web.fpp&version=4'`
改成
` api_url += '?type=web.fpp&version=4&appplt=flp&appid=pptv.flashplayer.vod&appver=3.4.2.12'`

**Comment by seahog004** [comment]
@airdge 修改后可以了，非常感谢！

**Comment by soimort** [comment]
Fixed by 56065c0a4881f89c93f840cf0afe92efd964701b.
